### DISCUSSION TITLE
ETA on the code release.

### DISCUSSION THREAD

**Discussion by JiamingSuen** [zju3dv/LoFTR#2] [General]
We plan to release the inference-only code and pre-trained model within the upcoming week, stay tuned. 
The entire codebase for data pre-processing, training and validation is under major refactoring and will be released around June.

**Comment by ducha-aiki** [comment]
Pytorch or tensorflow?

**Reply by JiamingSuen** [reply]
PyTorch, absolutely😉

**Reply by liuzw-cyy** [reply]
Thanks for your sharing！

**Comment by JiamingSuen** [comment]
We are happy to announce that the inference code of LoFTR is now publicly available! We also include an online demo and script to reproduce the testing results. The codebase for training is expected to be released before 2021-6-10.

All discussions (and potential bug reports) are welcome!

**Comment by xl-huo** [comment]
Thank you for your soild work!
Suppose you will release the ScanNet pre-processing/dataloader codebase, will there be MegaDepth's as well?

**Reply by JiamingSuen** [reply]
Yes, the training code on MegaDepth will be provided.

**Reply by xl-huo** [reply]
Thank you for the sharing!

**Reply by HUSTNO1WXY** [reply]
Linux or Windows?

**Comment by Devoe-97** [comment]
Would you release the training code today?

**Reply by JiamingSuen** [reply]
We are working on it and are a little behind schedule. We'll try to release the code before 6/15.

**Reply by Devoe-97** [reply]
Thank you for the sharing!
